roles/ssp/tasks/configure-common.yml

```diff
@@ -227,7 +227,24 @@
     - "config/config-metarefresh-{{ ssp_major_version }}.{{ ssp_minor_version }}.php.j2"
     - "config/config-metarefresh-{{ ssp_major_version }}.php.j2"
     - "config/config-metarefresh.php.j2"
-  when: "'metarefresh' in ssp_mods_enabled"
+  when: "'metarefresh' in ssp_mods_enabled and {{ ssp_major_version|float < 2}}"
```

Comment:
This type of condition causes the following warning:
```
[WARNING]: conditional statements should not include jinja2 templating
delimiters such as {{ }} or {% %}. Found: {{ ssp_major_version|float < 2}}
```
Could you try this instead?

```
when: ssp_major_version | int < 2
```

Reply:
done